src/i18n/index.ts

```diff
@@ -0,0 +1,61 @@
+import { langDe, langEn, langEs } from './langResources';
+
+import { ILangResource, ILangResourceCollection, IOSSpecific, LangResourceKeys } from '../models/i18n';
+
+const langResourceCollection: ILangResourceCollection = {
+  de: langDe,
+  en: langEn,
+  es: langEs,
+};
+
+export class LanguageResourceManager {
+
+  constructor(private resourceCollection?: ILangResourceCollection | {}) {
```

Comment:
Good idea. I also have to look into making `langResourceCollection` an independent file. It's far from optimal but I didn't had much time to iterate on it.